### PR TITLE
Feature/dom render callback

### PR DESCRIPTION
**Pull request by juliusb0205**
@ming-ancino 

complete functions

### CODE REVIEW

File: js/create_class.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+(function(namespace){
```
Comment (amanaloto) [inline js/create_class.js]: @juliusb0205 the `createClass` method is actually part of `Newton`. You can remove this file and add `createClass` to `newton.js` directly.

File: js/component.js [hunk 2]
```diff
@@ -33,17 +41,7 @@
...
-
-    for ( var key in props ){
-      if ( props.hasOwnProperty(key) ){
-        self.prototype[key] = props[key];
-      }
-    }
-
-    return self;
-  }
+    Newton.DOM.update(this);
```
Comment (amanaloto) [inline js/component.js]: @juliusb0205 would `namespace.Dom.update` work here?

File: js/create_class.js (hunk 1 above)
Comment (juliusb0205) [inline js/create_class.js]: Updated, removed create_class.js transferred method to newton.js

File: js/component.js (hunk 2 above)
Comment (juliusb0205) [inline js/component.js]: @amanaloto yes namespace.Dom.update works, i have updated this in new commit

**Pull request merged**